### PR TITLE
added author and software for MSMC

### PR DESCRIPTION
**Pull request by stschiff**
Hi all. As discussed with @jeromekelleher, I added myself to the author list and MSMC to the list of supported software. I'm afraid I think I won't get support in rarecoal done in time of resubmission. Hope that's OK for now. I'm honoured to be listed! Let me know if any other edits are needed.

### CODE REVIEW

File: software-table.tex [hunk 1]
```diff
@@ -47,6 +47,8 @@
     Optimizes parameters in models of demographic history using SFS and linkage disequilibrium statistics
     \citep{jouganous2017inferring,ragsdale2019models}. Models to be optimized can be specified in Demes.\\
 
+\MSMC & A script provided in the MSMC-tools repository (https://github.com/stschiff/msmc-tools) converts MSMC output to the demes format. Instructions are available in the Guide (https://github.com/stschiff/msmc-tools/blob/master/msmc-tutorial/guide.md)
```
Comment (grahamgower) [inline software-table.tex]: I think this needs a `\\` to end the line.

**Review by grahamgower — CHANGES_REQUESTED** [review]
Thanks @stschiff - I don't think this will build though, see comment below.

**Comment by jeromekelleher** [comment]
Can you add your grant acks to the Acknowledgement section too please @stschiff?

File: software-table.tex (hunk 1 above)
Comment (jeromekelleher) [inline software-table.tex]: Can you use ``\url{https://...}`` for the URLs please?

Comment (stschiff) [reply software-table.tex]: Ah yes. Fixed, sorry. Unfortunately I can't build this myself easily due to dependencies, can try later.

Comment (stschiff) [reply software-table.tex]: Done

**Comment by stschiff** [comment]
Yes, thanks, added acknowledgments, too.

**Comment by stschiff** [comment]
I've switched to Overleaf for tex-stuff, so my tex-fu on my local machine has been lost a bit. I've tried to build the paper myself, but so far encounter errors... so I rely on others to check whether it builds, for now. Sorry.

**Comment by stschiff** [comment]
Ah merged. OK, thanks!

**Pull request merged**